src/pilot/config_idrac.py

```diff
@@ -68,10 +69,16 @@ def parse_arguments():
     return parser.parse_args()
 
 
-def get_pxe_nic_fqdd(fqdd, model_properties, drac_client):
```

Comment:
I still don't understand why it is necessary to change this method signature.  Lines 76-81 below just repeat the code in the calling method.  We already have a drac_client created in the caller.  I can't see any reason why we wouldn't just pass that in here and use it.

Reply:
Oh, I see.  We're not planning on pushing those methods upstream.  We should change the code to use set_nic_settings() instead.